nodes/srl/srl.go

```diff
@@ -486,13 +485,12 @@ func (s *srl) createSRLFiles() error {
 			// as we will apply it over the top of a default config in the post deploy stage
 			return nil
 		}
+		cfgTemplate = string(c)
+	}
 
-		cfgTemplate := string(c)
-
-		err = s.GenerateConfig(dst, cfgTemplate)
-		if err != nil {
-			log.Errorf("node=%s, failed to generate config: %v", s.Cfg.ShortName, err)
-		}
+	err = s.GenerateConfig(cfgPath, cfgTemplate)
+	if err != nil {
+		log.Errorf("node=%s, failed to generate config: %v", s.Cfg.ShortName, err)
```

Comment:
@steiler this piece was needed I think.
without it. gen config func was never called when startup config was empty, hence no log message was printed

Reply:
Alright, then I think it is fine.